client/src/components/Visualizations/VisualizationHeader.vue

```diff
@@ -0,0 +1,35 @@
+<script setup lang="ts">
+import { faEye } from "@fortawesome/free-solid-svg-icons";
+
+import { type Plugin } from "@/api/plugins";
+import { absPath } from "@/utils/redirect";
+
+defineProps<{
+    plugin: Plugin;
+}>();
+</script>
+
+<template>
+    <div class="d-flex">
+        <div class="plugin-thumbnail mr-2">
+            <img v-if="plugin.logo" alt="visualization" :src="absPath(plugin.logo)" />
+            <icon v-else :icon="faEye" class="plugin-icon" />
```

Comment:
Consider using the `FontAwesomeIcon` for consistency, adjusting the icon size with the size prop.

```suggestion
            <FontAwesomeIcon v-else :icon="faEye" size="xl" class="p-1" />
```